### PR TITLE
update README and LOGO

### PR DESCRIPTION
**Pull request by pixiake**
### What does this PR do?
1. update README
2. update kubekey's logo

Signed-off-by: feng <guofeng@yunify.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
-* Install Kubernetes first, then deploy KubeSphere on it using [ks-installer](https://github.com/kubesphere/ks-installer)
+> KubeKey has passed [CNCF kubernetes conformance verification](https://www.cncf.io/certification/software-conformance/).
 
-> **Important:** If you have existing clusters, please refer to [ks-installer (Install KubeSphere on existing Kubernetes cluster)](https://github.com/kubesphere/ks-installer).
+There are three scenarios to use KubeKey.
 
-## Motivation
+* Install Kubernetes/k3s only
+* Install Kubernetes/k3s and KubeSphere together in one command
+* Install Kubernetes/k3s first, then deploy KubeSphere on it using [ks-installer](https://github.com/kubesphere/ks-installer)
```
Comment (Felixnoo) [inline README.md:19]: I think all `k3s` should be updated to `K3s`.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
 
-* Ansible-based installer has a bunch of software dependency such as Python. KubeKey is developed in Go language to get rid of the problem in a variety of environment so that increasing the success rate of installation.
-* KubeKey uses Kubeadm to install K8s cluster on nodes in parallel as much as possible in order to reduce installation complexity and improve efficiency. It will greatly save installation time compared to the older installer.
-* KubeKey supports for scaling cluster from allinone to multi-node cluster, even an HA cluster.
-* KubeKey aims to install cluster as an object, i.e., CaaO.
+> **Important:** If you have existing kubernetes clusters, please refer to [ks-installer (Install KubeSphere on existing Kubernetes cluster)](https://github.com/kubesphere/ks-installer).
```
Comment (Felixnoo) [inline README.md:21]: `kubernetes` -> `Kubernetes`

File: README.md [hunk 3]
```diff
@@ -43,6 +42,14 @@ There are three scenarios to use KubeKey.
 * **v1.22**: &ensp; *v1.22.1*
 > Looking for more supported versions [Click here](./docs/kubernetes-versions.md)
 
+### Container Manager
+* **Docker** / **Containerd** / **CRI-O** / **iSula**
```
Comment (Felixnoo) [inline README.md:46]: `Containerd` -> `containerd`

File: README.md [hunk 4]
```diff
@@ -43,6 +42,14 @@ There are three scenarios to use KubeKey.
...
+> `Kata Containers` can be set to automatically install and configure runtime class for it when the container manager is Containerd or CRI-O.
```
Comment (Felixnoo) [inline README.md:47]: `Containerd` ->  `containerd`

File: docs/developer-guide.md [hunk 5]
```diff
@@ -2,8 +2,12 @@
 
 KubeKey defines task pipelines for cluster operations such as installation, scaling, uninstallation, etc. And uses SSH and Kubernetes Api to perform corresponding tasks on hosts and cluster with host grouping and configuration management.
 
-![Image](img/KubeKey-Architecture.jpg?raw=true)
+![Image](img/KubeKey-Architecture.png?raw=true)
 
+* `Action`: The most basic unit. It represents one operation performed on one node;
+* `Task`: The one that manages `Action`. It contains fields such as `Action`, nodes, whether to retry, whether to execute in parallel, etc. In short, it represents executing an `Action` on the specified nodes as required;
```
Comment (Felixnoo) [inline docs/developer-guide.md:8]: > It contains fields such as `Action`, nodes, whether to retry, whether to execute in parallel, etc.

This sentence looks strange to me, especially the part after `Action`.

**Review by Felixnoo — CHANGES_REQUESTED** [review]
Please see my comments, thanks.

File: README.md [hunk 6]
```diff
@@ -1,25 +1,24 @@
...
 
 [![CI](https://github.com/kubesphere/kubekey/workflows/CI/badge.svg?branch=master&event=push)](https://github.com/kubesphere/kubekey/actions?query=event%3Apush+branch%3Amaster+workflow%3ACI+)
 
 > English | [中文](README_zh-CN.md)
 
-Since v3.0.0, [KubeSphere](https://kubesphere.io) changes the ansible-based installer to the new installer called KubeKey that is developed in Go language. With KubeKey, you can install Kubernetes and KubeSphere separately or as a whole easily, efficiently and flexibly.
+### 👋 Welcome to KubeKey!
 
-There are three scenarios to use KubeKey.
+KubeKey is an open source lightweight tool for deploying kubernetes clusters. It provides a flexible, rapid and convenient way to install Kubernetes/k3s only, both Kubernetes/k3s and KubeSphere, and related cloud-native add-ons. It is also an efficient tool to scale and upgrade your cluster.
```
[the remark this answers is not in the document]
Comment (pixiake) [reply README.md]: Done

File: README.md [hunk 7]
```diff
@@ -1,25 +1,24 @@
...
+
+In addition, KubeKey also supports customized Air-Gap package, which is convenient for users to quickly deploy clusters in offline environment.
```
[the remark this answers is not in the document]
Comment (pixiake) [reply README.md]: Yes, `air-gap` is a new feature.

File: README.md [hunk 8]
```diff
@@ -1,25 +1,24 @@
...
 
-* Install Kubernetes only
-* Install Kubernetes and KubeSphere together in one command
-* Install Kubernetes first, then deploy KubeSphere on it using [ks-installer](https://github.com/kubesphere/ks-installer)
+> KubeKey has passed [CNCF kubernetes conformance verification](https://www.cncf.io/certification/software-conformance/).
 
-> **Important:** If you have existing clusters, please refer to [ks-installer (Install KubeSphere on existing Kubernetes cluster)](https://github.com/kubesphere/ks-installer).
+There are three scenarios to use KubeKey.
```
[the remark this answers is not in the document]
Comment (pixiake) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (pixiake) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (pixiake) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (pixiake) [reply README.md]: Done

File: docs/developer-guide.md (hunk 5 above)
Comment (pixiake) [reply docs/developer-guide.md]: `Task` is the structure defined in the code, the following content describes some fields contained in this structure.

**Pull request merged**